packages/server/lib/util/print-run.ts

```diff
@@ -274,7 +287,17 @@ export function displaySpecHeader (name: string, curr: number, total: number, es
   }
 }
 
-export function renderSummaryTable (runUrl: string | undefined, results: any) {
+function maybeLogCloudRecommendationMessage (runs: CypressCommandLine.RunResult[], record?: boolean) {
+  if (!process.env.CI || env.get('CYPRESS_NO_COMMERCIAL_RECOMMENDATIONS') || record) {
```

Comment:
Shouldn't this be `!record`? Eg, if we are NOT recording, return. Otherwise, we might log the message.

Reply:
No, if the run is being recorded (if `--recorded` is passed, making `record = true`) then we want to exit early and **not** show the message.

We only want to suggest setting up on Cypress Cloud if they are **not already** recording the run. Maybe I could refactor the logic here to make things more clear?

Reply:
Right, gotcha, figured I was misreading this (I guess it's not immediately clear, or I just derped out... seems fine).